docker/Dockerfile

```diff
@@ -36,17 +41,21 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
     curl \
     smartmontools \
     tzdata \
+    && rm -rf /var/lib/apt/lists/* \
     && update-ca-certificates \
-    &&  case ${TARGETARCH} in \
-            "amd64")  S6_ARCH=amd64  ;; \
-            "arm64")  S6_ARCH=aarch64  ;; \
-        esac \
+    && case ${TARGETARCH} in \
+           "amd64")  S6_ARCH=amd64  ;; \
+           "arm64")  S6_ARCH=aarch64  ;; \
+       esac \
     && curl https://github.com/just-containers/s6-overlay/releases/download/v${S6VER}/s6-overlay-${S6_ARCH}.tar.gz -L -s --output /tmp/s6-overlay-${S6_ARCH}.tar.gz \
     && tar xzf /tmp/s6-overlay-${S6_ARCH}.tar.gz -C / \
     && rm -rf /tmp/s6-overlay-${S6_ARCH}.tar.gz \
-    && curl -L https://dl.influxdata.com/influxdb/releases/influxdb2-${INFLUXVER}-${TARGETARCH}.deb --output /tmp/influxdb2-${INFLUXVER}-${TARGETARCH}.deb \
+    curl -L https://dl.influxdata.com/influxdb/releases/influxdb2-${INFLUXVER}-${TARGETARCH}.deb --output /tmp/influxdb2-${INFLUXVER}-${TARGETARCH}.deb \
```

Comment:
missing `&&` before curl.

You should build this docker image locally and confirm it works.

Reply:
fixed. i rebuilt the docker image and it seems to work correctly. i verified that bookworm was used. i have a Vagrantfile that spins up one VM to build the docker image and then spins up another VM to actually run the build docker image. i can post that in another PR if you're interested.